include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -341,31 +346,31 @@ namespace internal
       }
 
       static void
-      access(gradient_type     &value,
-             const unsigned int vector_lane,
+      access(Tensor<1, spacedim, Number> &value,
+             const unsigned int           vector_lane,
              const unsigned int,
-             const scalar_gradient_type &shape_gradient)
+             const Tensor<1, spacedim, ScalarNumber> &shape_gradient)
```

Comment:
This should stay `scalar_gradient_type` (with `scalar_gradient_type` adjusted).

Reply:
Can you elaborate here? I tried to play with this parameter and it gets used in both `get_gradient`/`set_gradient` where I need it to be a tensor of entries, and in the interface you are pointing two. But that latter interfaces with the slow path and `FEValues`, where I believe the spacedim variant is fine. I might be missing something here, so I would appreciate a further comment.